### DISCUSSION TITLE
Week 12: Sept 3-6

### DISCUSSION THREAD

**Discussion by eeholmes** [SAFS-Varanasi-Internship/Summer-2024#26] [General]
@yifeihang @ShridharS19 This week and next should be focused on prepping a poster for AGU (and other events) and working on the final report. You can work on both at the same time. Content from the report can go into the poster.

* Sept 13 presentation to the other interns and mentors. Low key. 15 minutes each.
* min-Oct To Varanasi's, Tim Essington, Owen Hamel, other mentors and intern. Adapt the CoastWatch talk.
* November SAFS graduate student research symposium (poster session for undergrad intern).
* December AGU

Attached is a couple poster templates that would work for AGU. 6foot x 4foot (max) but 48 in x 36 in is common.
* https://docs.google.com/presentation/d/1PLJQae4xqQXyjaXFBw0fMDwdVBP-3RC-/edit?usp=sharing&ouid=103867251488366953420&rtpof=true&sd=true
* https://docs.google.com/presentation/d/158fH4guiTCcmnQTaSB1hFirqw44jzPcJ/edit?usp=sharing&ouid=103867251488366953420&rtpof=true&sd=true

You can google 'AGU poster examples' to see what typical posters are like.

**Comment by eeholmes** [comment]
Regarding writing the report. The past 2 years of interns have used Overleaf/LaTeX to write their reports. If you want to use that, then we should all link our UW accounts to Overleaf: https://www.overleaf.com/edu/uw. Then we can be collaborators without paying I believe.

Alternatively you can write the report in a Jupyter notebook. The markdown will understand LaTeX equations but the numbering might be minimal. But this allows you to use Python code to create figures and put them directly in the document.  This is how I normally write reports. You can output to PDF as you would with Overleaf.

**Comment by ShridharS19** [comment]
Progress + Check-In material @eeholmes 

https://docs.google.com/presentation/d/1dtDwF5E5lOn_8RBlFknvVyzekuGKoSxH/edit?usp=sharing&ouid=108414204333199215792&rtpof=true&sd=true

added a visualization for  ONet with our chlorophyll images, @yifeihang you can add one for UNet that is more based on our chlorophyll images if you can. Will continue editing tomorrow.

I will not make it to the intern check-in. For my progress, I would bring up my work at geo-smart HackWeek:
here is the poster we ended up with: https://docs.google.com/presentation/d/1djUnWQknhMInAaMahFHcml4FjA7v1rn0O8d-A74k6uA/edit?usp=sharing
My experience was great and I got to meet a lot of people of diverse experience that helped me reassess the whole project and gain new perspectives on it. Working with Ares I managed to get a deepONet example running from scratch which added a lot to my PINN results, my work with Gabriel allowed me to get intimately familiar with Yifei's work, setting up a gap-filling CNN from scratch. Andy and Robin helped me learn more about the bigger picture and scientific background, and Eli helped me restructure the project and master the fundamentals of the pipeline. I think the atmosphere there was very conducive to learning for any level, and everyone's help made it the most productive week of this internship.

**Comment by ShridharS19** [comment]
My internet connection i not the best so I will probably not be able to join the check-in today. Here are my updates:
https://docs.google.com/document/d/1F_WWDLWILU5Ly_IzkMiwYBEyggSt1H_ujbVU_DN6VWE/edit?usp=sharing
This is a copy of Yife's report doc with my own material, for a later merger.
I have taken after her format for CNNs and UNet with PINNs and DeepONet
 I have also added a lot of supporting images with some explanation pending for her section on the choice of environmental variables.
and the BIG result of today!
we finally have a comparison to a standard method(GB trees) for true error
![222222](https://github.com/user-attachments/assets/55b333a1-d6aa-4113-ba3d-a1208cd47fac)

wondering where and how to add this to the poster or report.
Also, in other news, I'm looking into Ares' latest commit(yesterday) with some advancement on the PINN stuff.  It might provide something for the report but most likely I will make sense of it after we draw up a draft.

**Reply by eeholmes** [reply]
@ShridharS19 Great news re the GB trees! Yes add to the poster as that was in the abstract. A figure showing the comparison to UNet would be good. Could be something like the line plots. Have UNet, PINN and GB in one figures
<img width="493" alt="image" src="https://github.com/user-attachments/assets/885febb7-b4ae-4e5c-b011-fbddd8000366">

**Reply by eeholmes** [reply]
@ShridharS19 So you need to run the GB Trees model with the 2020 test data and create predictions for each day. Same for PINN. And then you two can make a plot with all three lines so you can compare performance for 2020 across the 3 models.

**Reply by eeholmes** [reply]
So like this plot but a) with GB Trees added, b) only one y-axis so it is easy to compare the MADs, and c) level-3 fake clouds not level-4 gapfree comparison. So the line for U-Net will be the same as above. Add legend to id the 3 models
<img width="713" alt="image" src="https://github.com/user-attachments/assets/dce91f71-80fb-4778-a935-b60a51475f63">

**Comment by eeholmes** [comment]
@ShridharS19 @yifeihang If I am late tomorrow, get started without me and work on merging your material into one Google Doc (or overleaf or notebook).

**Comment by ShridharS19** [comment]
![unet vs xgboost](https://github.com/user-attachments/assets/4f599d5d-d37c-42e8-8e2a-a964fffbb9b4)

**Reply by ShridharS19** [reply]
will add pinn to this

**Reply by yifeihang** [reply]
I had a updated version of the level3/gapfree MAEs. Will resave them to npy files and push to my repo

**Reply by yifeihang** [reply]
pushed; should look like this:
![image](https://github.com/user-attachments/assets/070c8f63-39c7-4c03-9e22-dca05b64c4b4)
